docs/regmap/adi_regmap_spi_engine.txt

```diff
@@ -166,33 +166,40 @@ IRQ_MASK
 ENDREG
```

Comment:
Please change ENABLE default value 0x00000001 to 0x1

Reply:
Done here: https://github.com/analogdevicesinc/hdl/pull/1294/commits/9c5e3b37e5973b952a7d83267d5709effa4d17b8